City-of-Pori/pori-kada
# PORI-243&244&192: Search-bar in header and search-results page.

Search-bar in header and search-results page.

To test:
1. if you dont have solr running you need to provision and build site then check /admin/config/search/search_api
2. drush updb -y && drush fra -y && drush cc all
3. When you have search things running, make sure you have indexed content
4. Go to front page, confirm there is a search-icon next to language switch.
5. Click the icon and confirm search-field opens, if you click somewhere outside the search-block, the field should close.
6. Confirm searching works and takes you to the search-page, confirm search-page works and look ok
